packages/api/src/external/commonwell/document/shared.ts

```diff
@@ -14,7 +13,7 @@ export type CWDocumentWithMetriportData = Document & {
 };
 
 export function getFileName(patient: Patient, doc: Document): string {
-  const prefix = "document_" + makePatientOID("", patient.patientNumber).substring(1);
+  const prefix = "document_" + patient.id;
```

Comment:
This will likely result in files being redownloaded, although I failed to find files on `prod`'s S3 based on the previous logic.